brayns/Brayns.cpp

```diff
@@ -85,6 +86,10 @@ struct Brayns::Impl
         _parametersManager->parse(argc, argv);
         _parametersManager->print();
 
+        // Initialize Mesh loader
+        _meshLoader.reset(
```

Comment:
why pointer? can we not have a local meshloader everytime we need one? Does it have to be a permanent member of Brayns.cpp?

Reply:
We need a global meshloader indeed (and it's a pain). The reason behind it is that triangle faces use indices to vertices. When you load several meshes from different files, indices have to remain consistent. We could have many mesh loaders, but they would still have to aggregate at some point, so using a single one is the simple solution.

Reply:
the pointer is still not necessary. using proper initialization order in ctor should suffice